sdk/easm/azure-analytics-defender-easm/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 directory: specification/riskiq/Easm
 commit: 219fcd35385531e2af9c5b210ea195b8d05b393b
```

Comment:
use the commit id on main branch history https://github.com/Azure/azure-rest-api-specs/commits/main
e.g. https://github.com/Azure/azure-rest-api-specs/commit/f5f584c65071dc0306544a195fbcdff156023040